web/requirements.txt

```diff
@@ -1,6 +1,24 @@
-flask==1.1.2
-Flask-SQLAlchemy==2.4.1
-Flask-Migrate==2.5.3
+alembic==1.4.2
+bcrypt==3.1.7
+certifi==2020.4.5.1
+cffi==1.14.0
+click==7.1.2
+cryptography==2.9.2
+Flask==1.1.2
 Flask-Bcrypt==0.7.1
+flask-marshmallow==0.12.0
+Flask-Migrate==2.5.3
+Flask-SQLAlchemy==2.4.1
+itsdangerous==1.1.0
+Jinja2==2.11.2
+Mako==1.1.2
+MarkupSafe==1.1.1
+marshmallow==3.5.1
+marshmallow-sqlalchemy==0.22.3
+pycparser==2.20
 PyMySQL==0.9.3
-cryptography==2.9.2
\ No newline at end of file
+python-dateutil==2.8.1
+python-editor==1.0.4
+six==1.14.0
+SQLAlchemy==1.3.16
+Werkzeug==1.0.1
```

Comment:
Why did you add all of these explicitly? Most are sub-dependencies of the libraries already specified.

Reply:
I used pip freeze > requirements.txt to catch what to install for marshmallow but I can edit it to the shorten version if it is preferred

Reply:
since the sub dependencies get auto installed, let's leave them out. Just add the marshmallow here and it should be good.

Reply:
Ok I updated requirements.txt!